src/js/controller/controller.js

```diff
@@ -746,6 +731,13 @@ Object.assign(Controller.prototype, {
         this.getState = _getState;
         this.next = _nextUp;
         this.setConfig = (newConfig) => setConfig(_this, newConfig);
+        this.setItemIndex = _setItem;
+
+        // Program passthroughs
+        this.playVideo = _programController.playVideo.bind(_programController);
+        this.stopVideo = _programController.stopVideo.bind(_programController);
+        this.castVideo = _programController.castVideo.bind(_programController);
+        this.stopCast = _programController.stopCast.bind(_programController);
```

Comment:
Prefer arrow functions over `.bind` since we've seen examples of `function.prototype.bind` tampering.